src/99-appendices/08-coordinate-systems.md

```diff
@@ -70,18 +81,18 @@ detailed extensive descriptions of these EEG coordinate systems on the
 
 ## EEG Specific Coordinate Systems
 
-Restricted keywords for the `XXXCoordinateSystem` field in the
+Restricted keywords for the `<datatype>CoordinateSystem` field in the
 `coordsystem.json` file for EEG datasets:
 
--   `Captrak`: RAS orientation and the origin between the ears
+-   `CapTrak`: RAS orientation and the origin between the ears
```

Comment:
Looks like a backwards incompatible change. Is it just a typo being fixed? Do we need to worry about someone taking it too seriously in the past?

Reply:
yes, a typo fix. I haven't seen any datasets that use CapTrak yet --> and in fact, the list of restricted keywords for CoordSystems is not yet validated, see: https://github.com/bids-standard/bids-validator/issues/945

due to that missing coverage in the validator we currently have several datasets on openneuro that specify their coordsystem as e.g., "ARS", which is invalid of course. (see e.g., https://openneuro.org/datasets/ds002680/versions/1.0.0)

Reply:
agreed, being specified as "ARS" we still don't know how to relate electrode positions to the anatomy of the head, whereas "Other" and some textual description could have given more information.